_posts/2020-12-15-why-developer-should-know-owasp.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: Why a developer should know OWASP?
+author: riikkanen
+excerpt: OWASP stands for Open Web Application Security Project. Why should I be interested in OWASP as a developer? If I'm not a hacker or cybersecurity specialist, and we don't even handle any sensitive information in our project, can I just skip the OWASP? Well, the answer is "nope".
+tags: 
+- security
+- developers
+---
+
+OWASP stands for Open Web Application Security Project. Why should I be interested in OWASP as a developer? If I'm not a hacker or cybersecurity specialist, and we don't even handle any sensitive information in our project, can I just skip the OWASP? Well, the answer is "nope".

```

Comment:
Maybe add link to OWASP?
https://owasp.org/

Reply:
added